bluesky_widgets/_matplotlib_axes.py

```diff
@@ -137,8 +149,10 @@ def _add_image(self, image_spec):
         array = image_spec.func(run)
 
         # Initialize artist with currently-available data.
-        artist = self.axes.imshow(array, label=image_spec.label, **image_spec.style)
+        artist = self.axes.imshow(array, label=image_spec.label, **image_spec.style, origin='lower')
```

Comment:
Minor thing: let's put the unpacking parameter, `**image_spec.style`, last. Otherwise this leads me to wonder if there is something special going on with precedence (but there isn't) and "looks funny".